pycm/pycm_util.py

```diff
@@ -409,3 +409,111 @@ def statistic_recommend(classes, P):
     if binary_check(classes):
         return BINARY_RECOMMEND
     return MULTICLASS_RECOMMEND
+
+
+def matrix_combine(matrix_1, matrix_2):
+    """
+    Return the combine of two Confusion matrices.
+
+    :param matrix_1: First matrix that is going to be combined.
+    :type matrix_1: dict
+    :param matrix_2: Second matrix that is going to be combined.
+    :type matrix_2: dict
+    :return: combine of two matrices as a dict of dicts
```

Comment:
The combination of two matrices as a dict of dicts

Reply:
Fixed in https://github.com/sepandhaghighi/pycm/pull/344/commits/b5eb3bc08c02b59bd9a452cfa786c0ca9dd412f0